[2025-02-05T13:25:02.257+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-05T13:25:02.277+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_dbt-model_sales.model_sales.dim_vendedor_seed manual__2025-02-05T13:24:50+00:00 [queued]>
[2025-02-05T13:25:02.293+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_dbt-model_sales.model_sales.dim_vendedor_seed manual__2025-02-05T13:24:50+00:00 [queued]>
[2025-02-05T13:25:02.294+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-02-05T13:25:02.316+0000] {taskinstance.py:2330} INFO - Executing <Task(DbtSeedLocalOperator): model_sales.dim_vendedor_seed> on 2025-02-05 13:24:50+00:00
[2025-02-05T13:25:02.325+0000] {standard_task_runner.py:63} INFO - Started process 3928 to run task
[2025-02-05T13:25:02.330+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_dbt-model_sales', 'model_sales.dim_vendedor_seed', 'manual__2025-02-05T13:24:50+00:00', '--job-id', '113', '--raw', '--subdir', 'DAGS_FOLDER/dag_dbt-model_sales.py', '--cfg-path', '/tmp/tmp_rij_uid']
[2025-02-05T13:25:02.332+0000] {standard_task_runner.py:91} INFO - Job 113: Subtask model_sales.dim_vendedor_seed
[2025-02-05T13:25:02.431+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_dbt-model_sales.model_sales.dim_vendedor_seed manual__2025-02-05T13:24:50+00:00 [running]> on host 98025f4a542d
[2025-02-05T13:25:02.636+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_dbt-model_sales' AIRFLOW_CTX_TASK_ID='model_sales.dim_vendedor_seed' AIRFLOW_CTX_EXECUTION_DATE='2025-02-05T13:24:50+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-05T13:24:50+00:00'
[2025-02-05T13:25:02.645+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-05T13:25:06.630+0000] {local.py:220} INFO - dbtRunner is available. Using dbtRunner for invoking dbt.
[2025-02-05T13:25:06.632+0000] {local.py:436} INFO - Cloning project to writable temp directory /tmp/tmp649xi_5r from /usr/local/airflow/dags/dbt/localiza_dw
[2025-02-05T13:25:06.643+0000] {local.py:447} INFO - Partial parse is enabled and the latest partial parse file is /tmp/cosmos/dag_dbt-model_sales__model_sales/target/partial_parse.msgpack
[2025-02-05T13:25:06.654+0000] {config.py:348} INFO - Profile caching is enable.
[2025-02-05T13:25:06.670+0000] {base.py:84} INFO - Using connection ID '***' for task execution.
[2025-02-05T13:25:06.672+0000] {config.py:328} INFO - Profile found in cache using profile: /tmp/cosmos/profile/47873256e3df2d6067f929148b3671caee72f33396318ef25aae33418849eddf/profiles.yml.
[2025-02-05T13:25:06.673+0000] {local.py:408} INFO - Trying to run dbtRunner with:
 ['seed', '--vars', '{"PROD": P003,                                 "LIMITE_QUANTIDADE": 500,                                 "DATA_INICIO": 1999-01-01,                                 "DATA_FIM": 2025-02-05}', '--models', 'dim_vendedor', '--project-dir', '/tmp/tmp649xi_5r', '--profiles-dir', '/tmp/cosmos/profile/47873256e3df2d6067f929148b3671caee72f33396318ef25aae33418849eddf', '--profile', '***', '--target', 'prod']
 in /tmp/tmp649xi_5r
[2025-02-05T13:25:06.703+0000] {logging_mixin.py:188} INFO - 13:25:06  Running with dbt=1.6.1
[2025-02-05T13:25:06.873+0000] {logging_mixin.py:188} INFO - 13:25:06  Registered adapter: postgres=1.6.1
[2025-02-05T13:25:06.909+0000] {logging_mixin.py:188} INFO - 13:25:06  Unable to do partial parsing because config vars, config profile, or config target have changed
[2025-02-05T13:25:06.911+0000] {logging_mixin.py:188} INFO - 13:25:06  Unable to do partial parsing because profile has changed
[2025-02-05T13:25:06.913+0000] {logging_mixin.py:188} INFO - 13:25:06  Unable to do partial parsing because env vars used in profiles.yml have changed
[2025-02-05T13:25:08.695+0000] {logging_mixin.py:188} INFO - 13:25:08  Found 4 models, 1 snapshot, 4 seeds, 5 tests, 3 sources, 0 exposures, 0 metrics, 349 macros, 0 groups, 0 semantic models
[2025-02-05T13:25:08.700+0000] {logging_mixin.py:188} INFO - 13:25:08
[2025-02-05T13:25:08.912+0000] {logging_mixin.py:188} INFO - 13:25:08  Concurrency: 1 threads (target='prod')
[2025-02-05T13:25:08.914+0000] {logging_mixin.py:188} INFO - 13:25:08
[2025-02-05T13:25:08.923+0000] {logging_mixin.py:188} INFO - 13:25:08  1 of 1 START seed file dm_sales.dim_vendedor ................................... [RUN]
[2025-02-05T13:25:09.113+0000] {logging_mixin.py:188} INFO - 13:25:09  1 of 1 OK loaded seed file dm_sales.dim_vendedor ............................... [INSERT 20 in 0.19s]
[2025-02-05T13:25:09.134+0000] {logging_mixin.py:188} INFO - 13:25:09
[2025-02-05T13:25:09.136+0000] {logging_mixin.py:188} INFO - 13:25:09  Finished running 1 seed in 0 hours 0 minutes and 0.43 seconds (0.43s).
[2025-02-05T13:25:09.154+0000] {logging_mixin.py:188} INFO - 13:25:09
[2025-02-05T13:25:09.156+0000] {logging_mixin.py:188} INFO - 13:25:09  Completed successfully
[2025-02-05T13:25:09.157+0000] {logging_mixin.py:188} INFO - 13:25:09
[2025-02-05T13:25:09.159+0000] {logging_mixin.py:188} INFO - 13:25:09  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-02-05T13:25:09.950+0000] {local.py:138} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v10.json is above dbt-ol supported version 7. This might cause errors.
[2025-02-05T13:25:09.955+0000] {local.py:494} INFO - Inlets: []
[2025-02-05T13:25:09.956+0000] {local.py:495} INFO - Outlets: []
[2025-02-05T13:25:09.957+0000] {local.py:595} INFO - Assigning inlets/outlets without DatasetAlias
[2025-02-05T13:25:09.958+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2025-02-05T13:25:09.979+0000] {dag.py:3954} INFO - Setting next_dagrun for dag_dbt-model_sales to None, run_after=None
[2025-02-05T13:25:10.021+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-05T13:25:10.036+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_dbt-model_sales, task_id=model_sales.dim_vendedor_seed, run_id=manual__2025-02-05T13:24:50+00:00, execution_date=20250205T132450, start_date=20250205T132502, end_date=20250205T132510
[2025-02-05T13:25:10.090+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-02-05T13:25:10.121+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-05T13:25:10.124+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
